### DISCUSSION TITLE
Upgrade from 3.4.1 to 4.5.2

### DISCUSSION THREAD

**Discussion by pborgen** [emgucv/emgucv#528] [Q&A]
I am upgrading form  from 3.4.1 to 4.5.1 and running into some api issues.

This is code form 3.4.1. What would be the  equivalent in 4.5.1?
protected Dictionary<string, IImage> _Images = new Dictionary<string, IImage>();

**Comment by emgucv** [comment]
The IImage interface has been replaced with IInputArray in the 4.1.1 release:
https://www.emgu.com/wiki/index.php/Version_History#Emgu.CV-4.1.1

**Reply by pborgen** [reply]
Yeah I was looking at the diff. Are you willing to spend a couple hours with me to take a look at how I am migrating my code? Just wanted a more seasoned person to look at this. Willing to pay in fiat or crypto.... I am @borgenp on twitter

**Reply by emgucv** [reply]
Just sent you an email.
